cmd/controller/server_test.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"context"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	certUtil "k8s.io/client-go/util/cert"
+)
+
+func TestHttpCert(t *testing.T) {
```

Comment:
I see this is a new file so I assume there were no tests for the `/v1/cert.pem` endpoint until now? Thanks for adding one!

However, should there be a distinct test that replaces `cert` once the server is running (simulating a key rotation) and checks the latest cert is returned?

Reply:
thanks for encouraging me to write a proper test :-p